### ISSUE TITLE
Special tokens

### ISSUE THREAD

**Issue by antoniomastro1996** [google-research/text-to-text-transfer-transformer#624]
**Hi All!**

I was asking if there is a way to extend the set of special tokens that during the pre-training and the fine-tuning, I do not want the model mask.
For example in hugging face I can do the following: 
tokenizer.add_special_tokens({'additional_special_tokens': ['<special_token1>','<special_token2>'})

I also have another question, if, during the fine-tuning, I'm going to use "extra_id_X" token in the input sample, is T5 able to manage such a token like a special one?

**Thanks in adavance**
